tests/test_skip_tests.py

```diff
@@ -29,18 +29,28 @@
       must_not_contain:
         - "gorgonzola"
 
-- name: wall3_test
+- name: wall2_test
   command: bash -c "echo 'testing' > test3.txt"
   files:
     - path: test3.txt
       contains:
         - "kaas"
       must_not_contain:
         - "testing"
+
+- name: wall3_test
+  command: bash -c "exit 12"
+  files:
+    - path: test4.txt
+      contains:
+        - "content"
+      must_not_contain:
+        - "no_content"
+      md5sum: e583af1f8b00b53cda87ae9ead880224
 """)
 
 
 def test_skips(pytester):
     pytester.makefile(".yml", test=SKIP_TESTS)
     result = pytester.runpytest("-v").parseoutcomes()
-    assert {"failed": 4, "passed": 3, "skipped": 3} == result
+    assert {"failed": 5, "passed": 3, "skipped": 7} == result
```

Comment:
By having one combined result this makes the test prone to errors later. Two test units might fail in a way that the outcomes compensate for eachother. Also it makes it harder to adopt the test as you need to count all the units together to see what the expected numbers are. That is 15 tests to keep track of in your head. It is better to test the 3 units separately.